cabal.project

```diff
@@ -56,3 +44,7 @@ allow-newer: servant:*
 
 -- Required by trifecta (e.g. to allow mtl >=2.3)
 allow-newer: trifecta:*
+
+-- servant-0.20 does not yet support aeson-2.2
+--
+constraints: aeson <2.2
```

Comment:
Is this change unrelated to the prettyprinter vendoring?

Reply:
It is related because the bound on prettyprinter indirectly implied aeson <2.2